Automattic/jetpack
# AMP Sharing Support - Include all services

This code adds support for all the different sharing services that Jetpack makes available. It also include the ability to print the AMP page.

As per the discussion in [Issue 15399](https://github.com/Automattic/jetpack/issues/15399), We've opted to use the existing font based icons and override the default AMP styles. This gives us a consistent look between AMP and non-AMP modes.

Fixes #15399 
Fixes #14926 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This enhances an existing feature.
* P2 shortlink: https://wp.me/pbIAo1-w

#### Testing instructions:
1. Ensure the [AMP plugin](https://wordpress.org/plugins/amp/) is active
2. In AMP > Settings, select Transitional mode
3. Enable sharing in Jetpack > Settings > Sharing or Settings > Sharing
4. Visit a post on the front end and check that social icons are enabled
5. Add `?amp` to the end URL and make sure the AMP icons display as per step 4
6. Expected: Sharing icons look the same on the AMP and non-AMP versions of the front end website.

Non-AMP:
<img width="1280" alt="Screen Shot 2020-04-20 at 5 08 01 PM" src="https://user-images.githubusercontent.com/49400/79767435-a0080780-8329-11ea-9edb-4031ce43c1c5.png">

AMP:
<img width="1280" alt="Screen Shot 2020-04-20 at 5 08 08 PM" src="https://user-images.githubusercontent.com/49400/79767455-a5fde880-8329-11ea-89eb-4783e4d7c64a.png">
